packages/shared/src/styles/components/buttons.css

```
@@ -172,11 +192,7 @@
 
   & [aria-pressed="true"] ~ label {
     color: var(--button-pressed-color);
-  }
-
-  & :hover ~ label, & .hover ~ label, & :focus:focus-visible ~ label {
-    color: var(--button-hover-color);
-  }
+  }  
```

Just reverting the placement here.

```suggestion
  }
  
  & :hover ~ label, & .hover ~ label, & :focus:focus-visible ~ label {
    color: var(--button-hover-color);
  }
```